general/server/media/shows.md

```diff
@@ -5,19 +5,19 @@ title: Shows
 
 # Shows
 
-The most common naming scheme for shows is categorizing the files by series and then season. Another common method is simply using series folders, especially for shows that are organized by air date and those without seasons. Adding the year at the end in parentheses will yield the best results when scraping metadata.
+The most common naming scheme for shows is categorizing the files by series and then season. Another common method is simply using series folders, especially for shows that are organized by air date and those without seasons. Adding the year at the end in parentheses along with a media DB ID will yield the best results when scraping metadata. The two DBs supported by Jellyfin are [IMDB](https://www.imdb.com/) and [TMBD](https://www.themoviedb.org/). Media management software such as [Sonarr](https://sonarr.tv/) can be configured to add these IDs in to the folder names. The examples below show the ID syntax `[idtype-ID]`.
 
 ```txt
 Shows
-├── Series (2010)
+├── Series (2010) [imdbid-ID]
```

Comment:
This makes it seem like you *have* to put the IDs in the file names for proper naming, which isn't the case.

Sidenote: If anything, it may lead to more false positives for people, as it's more stuff for the filename parser to deal with and try to recognize. Hence why I'd argue that this feature should be removed altogether and replaced with support for [Kodi's Parsing NFOs](https://kodi.wiki/view/NFO_files/Parsing).

Reply:
Removed that change. Although I'd argue that parsing `[imdbid-%ID%]` should be relatively doable and bug free. I also thought Jellyfin already parsed NFOs? Also NFOs can contain more data that can become stale and some people would prefer for Jellyfin not to have write access to the media files. Although maybe NFOs make more sense for larger collections as I imagine scanning them becomes quicker?